### PR TITLE
Adds Kit command line argument support

### PR DESCRIPTION
**Pull request by kellyguo11**
# Description

This change adds the option to pass command line arguments directly to OV kit. This avoids the need of having to modify the app files to change settings for OV.

## Type of change

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab/docs/CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changelog
 ---------
 
+0.27.1 (2024-10-21)
+~~~~~~~~~~~~~~~~~~~
+
+Added
+^^^^^
+
+* Added ``--ov_args`` to :class:`~omni.isaac.lab.app.AppLauncher` to allow passing command line arguments directly to Omniverse.
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/docs/CHANGELOG.rst]:
```suggestion
* Added ``--kit_args`` to :class:`~omni.isaac.lab.app.AppLauncher` to allow passing command line arguments directly to Omniverse Kit SDK.
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py [hunk 2]
```diff
@@ -185,6 +185,10 @@ def add_app_launcher_args(parser: argparse.ArgumentParser) -> None:
           * If headless and enable_cameras are False, the experience file is set to ``isaaclab.python.kit``.
           * If headless is True and enable_cameras is False, the experience file is set to ``isaaclab.python.headless.kit``.
 
+        * ``ov_args`` (str): Optional command line arguments to be passed to Omniverse Kit directly.
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py]:
```suggestion
        * ``kit_args`` (str): Optional command line arguments to be passed to Omniverse Kit directly.
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py [hunk 3]
```diff
@@ -271,6 +275,15 @@ def add_app_launcher_args(parser: argparse.ArgumentParser) -> None:
                 " it is resolved relative to the `apps` folder in Isaac Sim and Isaac Lab (in that order)."
             ),
         )
+        arg_group.add_argument(
+            "--ov_args",
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py]:
```suggestion
            "--kit_args",
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py [hunk 4]
```diff
@@ -271,6 +275,15 @@ def add_app_launcher_args(parser: argparse.ArgumentParser) -> None:
...
+            type=str,
+            default="",
+            help=(
+                "Command line arguments for Omniverse Kit as a string separated by a space delimiter."
+                ' Example usage: --ov_args "--ext-folder=/path/to/ext1 --ext-folder=/path/to/ext2"'
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py]:
```suggestion
                ' Example usage: --kit_args "--ext-folder=/path/to/ext1 --ext-folder=/path/to/ext2"'
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py [hunk 5]
```diff
@@ -271,6 +275,15 @@ def add_app_launcher_args(parser: argparse.ArgumentParser) -> None:
                 " it is resolved relative to the `apps` folder in Isaac Sim and Isaac Lab (in that order)."
             ),
         )
+        arg_group.add_argument(
+            "--ov_args",
+            type=str,
+            default="",
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py:281]: Can we do something better than a string here?

https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse

```suggestion
            type=str,
            default="",
```

**Review by Dhoeller19 — APPROVED** [review]

**Pull request merged**